src/views/LocationInputForm.jsx

```diff
@@ -318,16 +318,14 @@ export const LocationInputForm = connect(
                         value: e.target.checked,
                       });
                     }}
-                  />
-                  <Label
-                    className="msab-has-text-grey margin-left-10"
-                    htmlFor="tos"
                   >
-                    I agree to the{' '}
-                    <a href="tos" target="_new">
-                      terms of service
-                    </a>
-                  </Label>
+                    <span className="msab-has-text-grey margin-left-10">
+                      I agree to the{' '}
+                      <a href="tos" target="_new">
```

Comment:
btw, target should be `"_blank"` (`"_new"` isn't a valid attribute) but it probably works because browsers are cool like that 
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target

Reply:
Ah, good call - I'll fix that quick.